mne/epochs.py

```diff
@@ -404,6 +404,42 @@ def decimate(self, decim, copy=False):
             epochs.times = epochs._raw_times[epochs._decim_slice]
         return epochs
 
+    def apply_baseline(self, baseline):
+        """Baseline correct epochs
+
+        Parameters
+        ----------
+        baseline : tuple of length 2
+            The time interval to apply baseline correction. (a, b) is the
+            interval is between "a (s)" and "b (s)". If a is None the beginning
+            of the data is used and if b is None then b is set to the end of
+            the interval. If baseline is equal to (None, None) all the time
+            interval is used.
+
+        Returns
+        -------
+        epochs : instance of Epochs
+            The baseline-corrected Epochs object.
+
+        Notes
+        -----
+        Baseline correction can be done multiple times.
+
+        .. versionadded:: 0.10.0
+        """
+        if not isinstance(baseline, tuple):
+            raise ValueError('`baseline=%s` is an invalid argument.'
+                             % baseline)
+
+        verbose = self.verbose
```

Comment:
does the verbose decorator then changes the verbose status if it were set to True in the method call?
my initial thinking was that since it is a method that it should use the verbose setting of the object.